src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	private String block;
+	
+	public Block(String _block){
+		this.block = _block;
```

Comment:
As mentioned in the tutorial, this naming style does not match what is prescribed in the coding standard.